frontend/src/Pages/Student/StudentDashBoard/StudentDashboard.jsx

```
@@ -1,92 +1,92 @@
                     </div>
-                ) : (
-                    <p>Loading student data...</p>
-                )}
+                </div>
+
+                {/* Logout Button */}
+                <div className="mt-6 flex justify-center">
+                    <button
+                        onClick={logout}
+                        className="px-5 py-2 flex items-center gap-2 bg-red-600 text-white font-semibold rounded-lg shadow-md hover:bg-red-700 transition"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)